### ISSUE TITLE
`EDGEDRIVER_VERSION=stable` tries to download previous stable instead of latest and fails

### ISSUE THREAD

**Issue by AriPerkkio** [webdriverio-community/node-edgedriver#239]
We are using [`browser-actions/setup-edge`](https://github.com/browser-actions/setup-edge) to setup Edge before running Webdriverio. `setup-edge` downloads newer version than `webdriverio` when both are passed `"stable"` as version.

- Webdriver: https://msedgedriver.azureedge.net/LATEST_STABLE -> `121.0.2277.112`
- Setup Edge: https://edgeupdates.microsoft.com/api/products -> `{ "Product": "Stable", "ProductVersion": "121.0.2277.113"}`
  - https://github.com/browser-actions/setup-edge/blob/af86699131c05a7e6ebf6dd2be7ca2a5cf41eef9/src/edge_api.ts#L111

The version that Webdriverio is trying to download returns 404. Replace the `112` with `113` and it works. Maybe `112` is already removed?
- https://msedgedriver.azureedge.net/121.0.2277.112/edgedriver_linux64.zip

Maybe this is related to https://github.com/webdriverio-community/node-edgedriver/commit/9374e6945c7f495205a83f8e2144cf59aea6aad8#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199R6 (@christian-bromann 👋)?

Failed CI run with `logLevel: "debug"`: https://github.com/vitest-dev/vitest/actions/runs/7875361078/job/21487100645?pr=5186

<hr />

```yml
browser-actions/setup-edge@v1
  with:
    edge-version: stable
```

```
Attempting to download Edge stable...
Acquiring stable (121.0.2277.113) from https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-stable/microsoft-edge-stable_121.0.2277.113-1_amd64.deb
```

```yml
pnpm run test:browser:webdriverio
  env:
    EDGEDRIVER_VERSION: stable
```

```
2024-02-12T17:13:47.826Z INFO edgedriver: Downloading Edgedriver from https://msedgedriver.azureedge.net/121.0.2277.112/edgedriver_linux64.zip
```

**Comment by christian-bromann** [comment]
@AriPerkkio thanks for raising the issue.

I think it makes sense to switch to `https://edgeupdates.microsoft.com/api/products` then for fetching the latest stable version. Wdyt?

**Comment by christian-bromann** [comment]
fixed in `v5.3.10`

**Comment by AriPerkkio** [comment]
Thanks for quick actions @christian-bromann! 

The fix seems to work nicely:

> INFO edgedriver: Starting Microsoft Edge WebDriver 121.0.2277.113
